### PR TITLE
Late commands vs done

### PR DESCRIPTION
**Pull request by dbungert**

### CODE REVIEW

File: subiquity/common/types/__init__.py [hunk 1]
```diff
@@ -91,6 +91,8 @@ class ApplicationState(enum.Enum):
     UU_RUNNING = enum.auto()
     UU_CANCELLING = enum.auto()
```
Comment (ogayot) [inline subiquity/common/types/__init__.py:92]: I wonder if we want to drop this as well.

**Review by ogayot — COMMENTED** [review]
I would like to separate the removal of the unattended-upgrades cancellation from the late-commands status change. I will split the commits into two accordingly.

File: subiquity/common/types/__init__.py (hunk 1 above)
Comment (ogayot) [reply subiquity/common/types/__init__.py]: @d-loose Do you see a problem with removing one this value from the enumeration?

Comment (d-loose) [reply subiquity/common/types/__init__.py]: Not at all, we're not using it anywhere in the desktop installer

**Comment by ogayot** [comment]
So I just tested this PR against the desktop installer. Although the meta/status reports `{"state": "LATE_COMMANDS"}`, the UI would still show the Installation complete screen (and the restart now button). So it requires more work.

File: subiquity/common/types/__init__.py [hunk 2]
```diff
@@ -86,10 +86,11 @@ class ApplicationState(enum.Enum):
     WAITING = enum.auto()
     RUNNING = enum.auto()
 
-    # States reported while unattended-upgrades is running.
-    # TODO: check if these should be dropped in favor of RUNNING.
+    # State reported while unattended-upgrades is running.
+    # TODO: check if this should be dropped in favor of RUNNING.
```
Comment (mwhudson) [inline subiquity/common/types/__init__.py:90]: They should only be distinct if a UI is going to care about the distinction and I think the way things are going the answer there is "no"?

File: subiquity/server/controllers/install.py [hunk 3]
```diff
@@ -758,8 +751,7 @@ async def run_unattended_upgrades(self, context, policy):
         apt_conf_path = Path(aptdir) / "zzzz-temp-installer-unattended-upgrade"
         apt_conf_path.write_bytes(apt_conf_contents)
         try:
-            self.unattended_upgrades_ctx = context
-            self.unattended_upgrades_cmd = await start_curtin_command(
+            uu_cmd = await start_curtin_command(
```
Comment (mwhudson) [inline subiquity/server/controllers/install.py:754]: This could be run_curtin_command I think?

Comment (ogayot) [reply subiquity/server/controllers/install.py]: Good point!

File: subiquity/common/types/__init__.py (hunk 2 above)
Comment (ogayot) [reply subiquity/common/types/__init__.py:90]: Currently, we have a distinction in the text UI so I'm tempted to leave it as is for now, wdyt?

Comment (mwhudson) [reply subiquity/common/types/__init__.py:90]: ok then!

**Review by Chris-Peterson444 — APPROVED** [review]

**Review by dbungert — COMMENTED** [review]
LGTM.  I can't Approve since I opened the PR ;)

**Pull request merged**